packages/client/hmi-client/src/components/widgets/tera-asset-enrichment.vue

```diff
@@ -132,7 +132,7 @@ const confirm = async () => {
 const sendForEnrichment = async () => {
 	// Build enrichment job ids list (profile asset, align model, etc...)
 	if (props.assetType === AssetType.Model) {
-		await modelCard(selectedResourceId.value);
+		await modelCard(props.assetId!, selectedResourceId.value);
```

Comment:
can probably do a check here for the assetId so we dont have the `!` if needed:
```
if(props.assetId && props.assetType === AssetType.Model) {
   await modelCard(props.assetId!, selectedResourceId.value);
   ...
}
  ```

Reply:
fixed